x-pack/platform/plugins/shared/ml/public/alerting/anomaly_detection_rule/register_anomaly_detection_rule.tsx

```diff
@@ -126,14 +126,14 @@ export function registerAnomalyDetectionRule(
 '{{#context.topInfluencers.length}}'
   Top influencers:
   '{{#context.topInfluencers}}'
-    '{{influencer_field_name}}' = '{{influencer_field_value}}' ['{{score}}']
+    '{{influencer_field_name}}-' = '{{influencer_field_value}}' ['{{score}}']
   '{{/context.topInfluencers}}'
 '{{/context.topInfluencers.length}}'
 
 '{{#context.topRecords.length}}'
   Top records:
   '{{#context.topRecords}}'
-    '{{function}}'('{{field_name}}') '{{by_field_value}}''{{over_field_value}}''{{partition_field_value}}' ['{{score}}']. Typical: '{{typical}}', Actual: '{{actual}}'
+    '{{function}}'('{{field_name}}-') '{{by_field_value}}''{{over_field_value}}''{{partition_field_value}}' ['{{score}}']. Typical: '{{typical}}', Actual: '{{actual}}'
```

Comment:
Would dropping the brackets `(` and `)` improve the formatting here?



`high_sum taxful_total_price- 'Wilhemina St. Ryan' [0]. Typical: $63.86, Actual: $200`

maybe better than:

`high_sum(taxful_total_price-) 'Wilhemina St. Ryan' [0]. Typical: $63.86, Actual: $200`